### ISSUE TITLE
Gap cursor extension not imported

### ISSUE THREAD

**Issue by robinbisping** [verbb/vizy#92]
At the moment, the gap cursor extension is not imported due to a typo.

[Line 46 in `VizyInput.vue`](https://github.com/verbb/vizy/blob/f3d1dcd8eb7cf3fbda708dfee606cd31adaf216b/src/web/assets/field/src/js/components/VizyInput.vue#L46) should be

```js
import Gapcursor from '@tiptap/extension-gapcursor';
```

instead of

```js
import Gapcursor from '@tiptap/extension-dropcursor';
```

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-3 as 1.0.8",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.0.9](https://github.com/verbb/vizy/releases/tag/1.0.9)
